tests/unit/style/parsing/image.rs

```diff
@@ -10,6 +10,22 @@ use style::values::specified::image::*;
 use url::Url;
 
 #[test]
+fn test_linear_gradient() {
+    // Parsing from the right
+    assert_roundtrip_with_context!(Image::parse, "linear-gradient(to left, red, green)",
+                                                   "linear-gradient(4.712389rad, red, green)");
+
+    // Parsing from the left
+    assert_roundtrip_with_context!(Image::parse, "linear-gradient(to right, red, green)",
+                                                 "linear-gradient(1.5707964rad, red, green)");
+
+    // Parsing without <angle> and <side-or-corner>
+    assert_roundtrip_with_context!(Image::parse, "linear-gradient(red, green)",
+                                                 "linear-gradient(3.1415927rad, red, green)");
+
```

Comment:
There is a [rustfmt](https://github.com/rust-lang-nursery/rustfmt) as far as I know